info/available-rules.md

```diff
@@ -1,131 +1,131 @@
+| 1    | 1.1.1    | GENERIC_NAME                              | Check: warns if a generic name contains more than 1 letter (capital). It can be followed by numbers, for example: T12, T<br>Fix:                                                                                                                                   | yes | no                                                                                                                                                                                                                         | Recursively update usages of this identifier in the project                                                                                                                                                                   |
+| 1    | 1.1.1    | VARIABLE_HAS_PREFIX                       | Check: warns if a variable has a prefix (like mVariable or M_VARIABLE), generally it is a bad code style (Android - is the only exception<br>Fix: no fix is available to automatically create a name                                                      | no  | no                                                                                                                                                                                                                         | Maybe we can fix it, but I do not see any sense for it                                                                                                                                                                      |
+| 1    | 1.2.1    | PACKAGE_NAME_MISSING                      | Check: warns if the package name is missing in a file<br>Fix: automatically adds a package directive with the name that starts from the domain name (example - com.huawei) and contains the real directory                                                      | yes | no                                                                                                                                                                                                                         | Recursively fix all imports in the project.<br>Fix the directory where the code is stored.<br>Make this check isolated from the domain name addition.                                                                                  |
```

Comment:
> The check warns if a file is missing the package declaration.

Reply:
1. OK
2. **Question**: What does the following sentence mean?
Maybe we can fix it, but I do not see any sense for it.

Reply:
2. This means that, technically, we can fix such issues automatically, but there's little sense in it. I believe you can remove thos whole sentence &mdash; it belongs to the bug tracker, **not** the user-visible documentation.

Reply:
OK. Removed.